### PR TITLE
docs(community): Update Gardening pages with slack links and videos

### PR DESCRIPTION
**Pull request by dnilasor**
I'd like to update the gardening area of the site. I've responded to comments from @plumpy on my previous PR with a few formatting and syntax edits, and by adding Slack links to the Slack channels listed in slack-map.md. 

I've also added a Minnaker-centric development environment setup video tutorial to the development environments page. This video is live in Spinnaker Youtube and is Spinnaker-branded. It's companion content to the Minnaker setup described in the existing content.

Also, folks, I FINALLY LEARNED HOW TO FOLLOW OUR COMMIT AND PR CONVENTIONS : D

@brian-armory  @dorbin thanks much!

### CODE REVIEW

**Comment by 317brian** [comment]
There's an untitled file in this commit that seems to be a mailchimp signup form.

File: community/gardening/slack-map.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
-|#gardening-gamers|Gamer chat|
-|#gardening-rave|Dance party|
-|#gardening-karaoke|Karaoke!|
-|#gardening-yoga-mindfulness|Scheduled yoga & mindfulness breaks|
-|#gardening-anonymous|Anonymous feedback|
-|#spinnaker-news|Share Spinnaker news & blog content for April newsletter|
-|#sig-documentation|Documentation SIG-hosted activities and chatter|
-|#sig-kubernetes|Kubernetes SIG-hosted chatter|
-|#sig-ui|UI SIG-hosted chatter|
+|[#gardening-ineedhelp](slack://channel?team={spinnakerteam}&id={gardening-ineedhelp})|Event helpdesk|
```
Comment (317brian) [inline community/gardening/slack-map.md]: These links don't work for me. They open Slack but don't switch channels or workspaces. I recommend just grabbing the channel URL's directly from Slack and using those by right clicking on a channel name in Slack and copying the channel link. You'll get the URL for a channel like gardening-ineedhelp, https://spinnakerteam.slack.com/archives/CURFZGL2E

**Comment by dnilasor** [comment]
Removed untitled file and fixed Slack links per @brian-armory thanks much!!

**Review by dorbin — APPROVED** [review]
LGTM, after you address Brian's comment about the Slack links. 

Thanks @dnilasor !

**Comment by dnilasor** [comment]
@dorbin , slack links updated. appreciate your review!!

File: community/gardening/slack-map.md (hunk 1 above)
Comment (dnilasor) [reply community/gardening/slack-map.md]: updated. thanks for this suggestion :woman-facepalming::skin-tone-3:

**Review by 317brian — APPROVED** [review]

**Pull request merged**